Game/Resources/TestScene.json

```diff
@@ -328,7 +410,7 @@
 			{
 				"x" : 0.0,
 				"y" : 0.0,
-				"z" : 0.0
+				"z" : 400.0
```

Comment:
Shouldn't controls be relative payer? Set to 0?

Reply:
This is cave's position. It shouldn't be 400 in z anyway but it has nothing to do with the player's position.